docs/develop/modules/injective/exchange/05_messages.md

```
@@ -441,3 +439,39 @@ type MsgBatchUpdateOrders struct {
+- `Sender` describes the creator of this msg.
+- `Grants` describes a list of grantees' addresses and grant amounts
+
+
+
+## Msg/ActivateStakeGrant
+
+`MsgActivateStakeGrant` is a message used to select/activate a stake grant for fee discount purposes.
+
+```go
+type MsgAuthorizeStakeGrants struct {
+	Sender  string 
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)